charon/src/translate/translate_functions_to_ullbc.rs

```diff
@@ -1471,16 +1471,21 @@ impl<'tcx, 'ctx, 'ctx1> BodyTransCtx<'tcx, 'ctx, 'ctx1> {
         mut self,
         rust_id: DefId,
         arg_count: usize,
+        item_meta: &crate::meta::ItemMeta,
```

Comment:
```suggestion
        item_meta: &ItemMeta,
```
and add a `use crate::meta::ItemMeta` at the start of the file.